net.solarnetwork.node.setup.web/web/WEB-INF/jsp/a/home.jsp

```diff
@@ -58,6 +70,8 @@
 </section>
 
 <section id="datum-activity" class="hide">
+	<h2>Datum Graphs</h2>
+	<p class="graphpoint"></p>
```

Comment:
I think a `<div>` is a better tag for this type of thing, saving `<p>` for actual text content. Also the class name should be quite clear in its intention. I think something like `datum-activity-charts` would be better.